src/main/java/org/jitsi/jicofo/recording/jibri/CommonJibriStuff.java

```diff
@@ -262,7 +263,10 @@ final synchronized IQ handleIQRequest(JibriIq iq)
                 // (so it isn't XMPPError.Condition.service_unavailable), so it
                 // must be that they're all busy.
                 logger.info("Failed to start a Jibri session, all Jibris were busy");
-                return IQ.createErrorResponse(iq, XMPPError.Condition.resource_constraint);
+                return ErrorResponse.create(
```

Comment:
What is the difference between `ErrorResponse.create` and `IQ.createErrorResponse` ?

Reply:
Ah, I see it's a new helper now.